src/graphql/search/datasets/normalize.ts

```diff
@@ -143,31 +143,33 @@ function getProperties(openApiData: any) {
   }
 }
 
-function getFacetOptions(facets: any, property: any) {
+function getFacetOptions(property: any, facets?: any) {
```

Comment:
`property` is `any` here but when passed to `getOptionsFromProperty` it gets a type of `PropertyType`. I assume we can use the same type here.

Reply:
Also `facets` is of type `Object` in other methods.